Basic/ClientDriven/Assets/Scripts/NetworkObjectSpawner.cs

```diff
@@ -0,0 +1,46 @@
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.Networking;
+using Random = System.Random;
```

Comment:
These two can be removed since Networking is unused, and Random isn't used in this class.

Reply:
Yes you are right!